### PR TITLE
CM-93: refactored constants at almost all modules

### PR DESCRIPTION
**Pull request by ozalexo**

### CODE REVIEW

File: src/layouts/partials/DrawerMainMenu/MenuTokensList/MenuTokensList.jsx [hunk 1]
```diff
@@ -9,14 +9,15 @@ import { Translate } from 'react-redux-i18n'
...
 import React, { PureComponent } from 'react'
 import { connect } from 'react-redux'
 import { drawerHide, drawerToggle } from 'redux/drawer/actions'
 import { DUCK_SESSION, logout } from '@chronobank/core/redux/session/actions'
 import { getBlockchainAddressesList } from '@chronobank/core/redux/session/selectors'
-import { SIDES_CLOSE_ALL, sidesPush } from 'redux/sides/actions'
-import MenuTokenMoreInfo, { MENU_TOKEN_MORE_INFO_PANEL_KEY } from '../MenuTokenMoreInfo/MenuTokenMoreInfo'
+import { sidesCloseAll, sidesPush } from 'redux/sides/actions'
+import MenuTokenMoreInfo from '../MenuTokenMoreInfo/MenuTokenMoreInfo'
+import { MENU_TOKEN_MORE_INFO_PANEL_KEY } from '../../../../redux/sides/constants'
```
Comment (ex1st0r) [inline src/layouts/partials/DrawerMainMenu/MenuTokensList/MenuTokensList.jsx]: Replace it with 'redux/sides/constants'

File: src/layouts/partials/DrawerMainMenu/DrawerMainMenu.jsx [hunk 2]
```diff
@@ -13,19 +13,20 @@ import classnames from 'classnames'
...
-import { SIDES_CLOSE_ALL, sidesPush } from 'redux/sides/actions'
+import { sidesCloseAll, sidesPush } from 'redux/sides/actions'
 import { modalsOpen } from 'redux/modals/actions'
 import UpdateProfileDialog from 'components/dialogs/UpdateProvideDialog/UpdateProfileDialog'
 import { getAccountAvatar, getAccountName } from '@chronobank/core/redux/persistAccount/utils'
 import { getWalletsLength } from '@chronobank/core/redux/wallets/selectors/wallets'
 import { getAccountProfileSummary } from '@chronobank/core/redux/session/selectors'
 import MenuAssetsManagerMoreInfo from './MenuAssetsManagerMoreInfo/MenuAssetsManagerMoreInfo'
-import { MENU_TOKEN_MORE_INFO_PANEL_KEY } from './MenuTokenMoreInfo/MenuTokenMoreInfo'
+import { MENU_TOKEN_MORE_INFO_PANEL_KEY } from '../../../redux/sides/constants'
```
Comment (ex1st0r) [inline src/layouts/partials/DrawerMainMenu/DrawerMainMenu.jsx]: Replace it with 'redux/sides/constants'

File: packages/core/redux/transactions/actions.js [hunk 3]
```diff
@@ -8,10 +8,10 @@ import { modalsOpenConfirmDialog } from '../../../core-dependencies/redux/modals
 import { ethereumProvider } from '../../../login/network/EthereumProvider'
```
Comment (ex1st0r) [inline packages/core/redux/transactions/actions.js]: dependency '@chronobank/login'

**Review by wAISw — APPROVED** [review]

File: src/redux/sides/reducer.js [hunk 4]
```diff
@@ -22,13 +32,13 @@ const initialState = {
...
     },
-    [NOTIFICATION_PANEL_KEY]: {
+    [ NOTIFICATION_PANEL_KEY ]: {
       component: NotificationContent,
       panelKey: NOTIFICATION_PANEL_KEY,
       isOpened: false,
       anchor: 'right',
     },
-    [MENU_ASSETS_MANAGER_PANEL_KEY]: {
+    [ MENU_ASSETS_MANAGER_PANEL_KEY ]: {
```
Comment (e-maximus) [inline src/redux/sides/reducer.js]: spaces

File: src/redux/sides/reducer.js [hunk 5]
```diff
@@ -22,13 +32,13 @@ const initialState = {
         width: 300,
       },
     },
-    [NOTIFICATION_PANEL_KEY]: {
+    [ NOTIFICATION_PANEL_KEY ]: {
```
Comment (e-maximus) [inline src/redux/sides/reducer.js]: !

**Review by e-maximus — APPROVED** [review]

**Pull request merged**